packages/terra-embedded-content-consumer/tests/nightwatch/providers/MessagingProvider.jsx

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
This is testing XFC... which we have thorough tests for. I recommend removing this test file